src/objects.js

```diff
@@ -1,41 +1,50 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age
+  };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property);
```

Comment:
in vscode you'll see that `hasOwnProperty` has been underlined by a red squiggly line. This is indicating that you shouldn't use this method in this test. Another way to check if the property exists is either use the [`in`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in) operator, e.g.
```
return property in object;
```
or to check if that property isn't equal to undefined, e.g.
```
return object[property] !== undefined;
```